<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup_progress</class>
 <widget class="QWidget" name="popup_progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loading</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="key_label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Helvetica&quot;;</string>
     </property>
     <property name="text">
      <string>Keyword</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="progress_label">
     <property name="text">
      <string>what is progress doing?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
